apps/prairielearn/src/lib/config.js

```diff
@@ -17,6 +17,19 @@ const ConfigSchema = z.object({
   postgresqlHost: z.string().default('localhost'),
   postgresqlPoolSize: z.number().default(100),
   postgresqlIdleTimeoutMillis: z.number().default(30_000),
+  postgresqlSsl: z
+    .union([
```

Comment:
I'm not sure I'm loving the use of `union`, just because I generally prefer things to have single clear types. We could allow `postgresqlSsl` to be `null` to mean "do not use SSL"? Or we could have a separate boolean like `postgresqlUseSsl`?

Reply:
Oh ok, I feel much better about this if it's mirroring the library types. Go for it.